### ISSUE TITLE
ERROR: for nginx-proxy  dictionary update sequence element ...

### ISSUE THREAD

**Issue by weeger** [JrCs/docker-letsencrypt-nginx-proxy-companion#295]
Hi,

Seeing https://github.com/docker/compose/issues/5349 with docker-compose version 1.17.0, the labels syntax change and create an error when launching companion.

```
ERROR: for nginx-proxy  dictionary update sequence element #0 has length 61; 2 is required
Traceback (most recent call last):
  File "bin/docker-compose", line 6, in <module>
  File "compose/cli/main.py", line 68, in main
  File "compose/cli/main.py", line 121, in perform_command
  File "compose/cli/main.py", line 952, in up
  File "compose/project.py", line 455, in up
  File "compose/parallel.py", line 70, in parallel_execute
ValueError: dictionary update sequence element #0 has length 61; 2 is required
Failed to execute script docker-compose
```

The solution may be to change the syntax, but what is the best way ?

```
    labels:
        com.github.jrcs.letsencrypt_nginx_proxy_companion.nginx_proxy: ???
```

**Comment by buchdag** [comment]
Wasn't it fixed in compose [1.17.1](https://github.com/docker/compose/releases/tag/1.17.1) ?

**Comment by buchdag** [comment]
No answers from issue creator for more than three months, closing.

**Comment by Egregors** [comment]
@buchdag Yes, this bug fixed in 1.17.1
